tests/cmds/test_users.py

```diff
@@ -263,3 +275,48 @@ def test_remove_user_role_raises_error_when_username_does_not_exist(
     result = runner.invoke(cli, command, obj=cli_state)
     assert result.exit_code == 1
     assert "User 'not_a_username@example.com' does not exist." in result.output
+
+
+def test_update_user_calls_update_user_with_correct_parameters(
+    runner, cli_state, update_user_success
+):
+    command = ["users", "update", "--user-id", "12345", "--email", "test_email"]
```

Comment:
Can you include all the args here so the test confirms they all get passed to the right spot?

Reply:
I wanted to test also that it properly passed `None` where no argument was given (since we don't want the method to accidentally update all unchanged fields to blank values). I've added additional tests to validate that all parameters are passed properly.